matlab/Glacier2/greeter/client.m

```diff
@@ -1,6 +1,8 @@
-% Copyright (c) ZeroC, Inc.
-
 function client(args)
+    %CLIENT An Ice client that connects to a Greeter server through a Glacier2 router.
+
+    % Copyright (c) ZeroC, Inc.
```

Comment:
That's the correct location for the copyright: after the main doc comment, and separated by a blank line so that it's not included in `help client`.

Before this PR, `help client` was showing the copyright and nothing else.